crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1831,6 +1829,18 @@ fn fixup_modules(modules: &mut [(DebugId, RawObjectInfo)]) {
     }
 }
 
+fn raw_object_info_equal(this: &RawObjectInfo, that: &RawObjectInfo) -> bool {
+    this.ty == that.ty
+        && this.code_id.as_ref().map(|id| id.parse::<CodeId>())
+            == that.code_id.as_ref().map(|id| id.parse())
+        && this.code_file == that.code_file
+        && this.debug_id.as_ref().map(|id| id.parse::<DebugId>())
+            == that.debug_id.as_ref().map(|id| id.parse())
+        && this.debug_file == that.debug_file
+        && this.image_addr == that.image_addr
+        && this.image_size == that.image_size
+}
```

Comment:
do you think it would make sense to move all the temporary comparisons code to a sub-module so that its easy to remove again and keeps the git history of this file a bit tidier?